netsim/data/types.py

```diff
@@ -764,6 +764,9 @@ def must_be_mac(value: typing.Any) -> dict:
 
   try:
     parse = netaddr.EUI(value)                                        # now let's check if we have a MAC address
+
+    if int(parse) & 0x010000000000:                                   # Check if the multicast bit is set
```

Comment:
One could use `bool(parse.words[0] & 0b01)` but we get the same result anyway ;)